api/v1alpha1/groupversion_info.go

```diff
@@ -15,8 +15,8 @@ limitations under the License.
 */
 
 // Package v1alpha1 contains API Schema definitions for the certification v1alpha1 API group
-//+kubebuilder:object:generate=true
-//+groupName=certification.redhat.com
+// +kubebuilder:object:generate=true
```

Comment:
This doesn't seem right? Do they still work with the space?

Reply:
I have no idea what is changing this in _this_ file **only**, but it should not have a space. The other files do not have a space and do not get changed.